README.md

```diff
@@ -8,11 +8,19 @@ _Please see the [documentation](https://virtualizarr.readthedocs.io/en/latest/)_
 
 ### Development Status and Roadmap
 
-VirtualiZarr is ready to use for many of the tasks that we are used to using kerchunk for, but the most general and powerful vision of this library can only be implemented once certain changes upstream in Zarr have occurred.
+VirtualiZarr version 1 focused on (mostly) achieving feature parity with kerchunk, providing an easier way to manipulate kerchunk references in memory and generate kerchunk reference files on disk.
```

Comment:
```suggestion
VirtualiZarr version 1 achieves feature parity with kerchunk, providing an easier way to manipulate kerchunk references in memory and generate kerchunk reference files on disk.
```

What features are you referring to you here when you say "achieves feature parity with kerchunk"? I see VirtualiZarr version 1's value is providing typed interfaces to chunk references, which are not present in kerchunk, and those interfaces are more flexible than the kerchunk reference store. But most of the "features" are still kerchunk under the hood, right? Maybe that's what you mean by parity?

Reply:
I have [this table in the docs](https://virtualizarr.readthedocs.io/en/latest/faq.html#how-do-virtualizarr-and-kerchunk-compare) to track all the features. I should link to that here.

"(mostly) achieves feature parity" is obviously very arbitrary, but for me the most important part was to be able to open a typical netcdf file (i.e. with time encoding), concatenate using xarray, save to kerchunk refs, and roundtrip.

Perhaps it would be clearer (and more charitable) to say "(mostly) achieves feature parity with kerchunk's logic for combining datasets".

> But most of the "features" are still kerchunk under the hood

I'm not sure I would say "most", but yes. To completely decouple from kerchunk all we need now are Sean's PR and upstreaming the zarr chunk manifest stuff.